src/explorer/SiteTreeItem.ts

```diff
@@ -124,6 +136,33 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             });
     }
 
+    private async enableScmDoBuildDuringDeploy(fsPath: string, runtime: string): Promise<void> {
+        const yesButton: MessageItem = { title: 'Yes' };
+        const dontShowAgainButton: MessageItem = { title: "Don't show again", isCloseAffordance: true };
```

Comment:
This button isn't the close affordance. There should still be a 'close' button that means "I don't want to enable scm right now, but warn me again next time"